### PR TITLE
John Roemer PR

### PR DESCRIPTION
**Pull request by jtrtj**
got up to highest_scoring_word method

### CODE REVIEW

**Comment by jtrtj** [comment]
got up to highest_scoring_word method

File: lib/scrabble.rb [hunk 1]
```diff
@@ -15,4 +15,48 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    if word == nil
+      return nil
+    else
+    count_score(word).flatten.sum
```
Comment (mikedao) [inline lib/scrabble.rb:23]: Watch your indentation

File: lib/scrabble.rb [hunk 2]
```diff
@@ -15,4 +15,48 @@ def point_values
...
+    end
+  end
+
+  def score_with_multipliers(word, position, multiplier = 1)
+    score = count_score(word).flatten
+    array = score.zip(position).map do |s,p|
+      s * p
```
Comment (mikedao) [inline lib/scrabble.rb:30]: dont use single letter variables

File: lib/scrabble.rb (hunk 1 above)
Comment (mikedao) [inline lib/scrabble.rb:23]: Why are we flattening?

**Pull request closed without merging**